transaction_receipt_query.go

```diff
@@ -137,7 +137,7 @@ func _TransactionReceiptQueryShouldRetry(request _Request, response _Response) _
 	switch Status(response.query.GetTransactionGetReceipt().GetReceipt().GetStatus()) {
 	case StatusBusy, StatusUnknown, StatusOk, StatusReceiptNotFound, StatusRecordNotFound:
 		return executionStateRetry
-	case StatusSuccess:
+	case StatusSuccess, StatusIdenticalScheduleAlreadyCreated:
```

Comment:
@failfmi We should be treating all receipt statuses that we don't retry on as successful.
https://github.com/hashgraph/hedera-sdk-go/pull/327